### PR TITLE
Add keyboard: Xtrfy K4 TKL RGB, ISO only, no RGB support

### PR DESCRIPTION
**Pull request by humanplayer2**
Adds support for _Xtrfy K4 TKL RGB_ board. ISO only, no RGB support.

I have been unable to get rid of all RGB code to make a clean `plain` version.

### CODE REVIEW

**Comment by humanplayer2** [comment]
@dexter93 has now helped me clean out the RGB LED things.

**Comment by humanplayer2** [comment]
@dexter93 suggested this should be based on `keychron/k8` from `sn32_develop` instead of `sn32_master`.  This has now been done.

File: keyboards/xtrfy/k4/plain/config.h [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
```
Comment (dexter93) [inline keyboards/xtrfy/k4/plain/config.h]: this is now stale. please remove

Comment (humanplayer2) [reply keyboards/xtrfy/k4/plain/config.h]: Done. My apologies for the slow reaction.

**Comment by humanplayer2** [comment]
@dexter93 , I think I made the changes you requested.

**Pull request merged**